convert/lib/convert.py

```diff
@@ -0,0 +1,545 @@
+                    "system:nodes",
+                ],
+            }
+        ]
+
+        tfvars = {
+            "deploy_id": cdkconfig["name"],
+            "region": cdkconfig["aws_region"],
+            "grandfathered_creation_role": self.stacks["eks_stack"]["resources"]["eksCreationRole"],
+            "tags": {**cdkconfig["tags"], "domino-deploy-id": cdkconfig["name"]},
+            "vpc_id": cdkconfig["vpc"]["id"]
+            if not cdkconfig["vpc"]["create"]
+            else self.stacks["vpc_stack"]["resources"]["VPC"],
+            "public_subnet_ids": get_subnet_ids("Public"),
+            "private_subnet_ids": get_subnet_ids("Private"),
+            "pod_subnet_ids": get_subnet_ids("Pod", ""),
+            "k8s_version": cdkconfig["eks"]["version"],  # We're trusting this is accurate
+            "ssh_key_path": self.args.ssh_key_path,
+            "number_of_azs": cdkconfig["vpc"]["max_azs"],
+            "route53_hosted_zone_name": reduce(
+                lambda cfg, k: cfg[k] if k in cfg else [""],
+                ["install", "overrides", "external_dns", "domain_filters"],
+                cdkconfig,
+            )[0],
+            "efs_backups": cdkconfig["efs"]["backup"]["enable"],
+            "efs_backup_schedule": cdkconfig["efs"]["backup"]["schedule"],
+            "efs_backup_cold_storage_after": cdkconfig["efs"]["backup"]["move_to_cold_storage_after"],
+            "efs_backup_delete_after": cdkconfig["efs"]["backup"]["delete_after"],
+            "efs_backup_force_destroy": cdkconfig["efs"]["backup"]["removal_policy"] == "DESTROY",
+            "eks_custom_role_maps": eks_custom_role_maps,
+        }
+
+        print(json.dumps(tfvars))
+
+    def clean_stack(self):
+        self.setup(full=True, no_stacks=self.args.resource_file)
+
+        if self.args.resource_file:
+            with open(self.args.resource_file) as f:
+                self.stacks = yaml.safe_load(f.read())
+
+        include_types = self.args.include_types or clean_categories
+
+        for t in include_types:
+            if t not in clean_categories:
+                raise ValueError(f"{t} not a valid category for `--include-types`. Should be: {clean_categories}")
+
+        lambda_safe = [
+            cdk_ids.lambda_function.value,
+            cdk_ids.iam_role.value,
+            cdk_ids.lambda_layerversion.value,
+            cdk_ids.stepfunctions_statemachine.value,
+        ]
+        nukey = {
+            "efs_stack": {
+                "(LogRetention|backuppostcreationtasks).*": lambda_safe,
+            },
+            "eks_cluster_stack": {
+                "(LogRetention|IsCompleteHandler|NodeProxyAgentLayer|OnEventHandler|Provider).*": lambda_safe,
+            },
+            "eks_kubectl_stack": {
+                "(Handler|KubectlLayer|ProviderframeworkonEvent).*": lambda_safe,
+            },
+            "eks_stack": {
+                f"(snapshot|chibipug7501ebscsi|{self.stack_name}DominoEcrRestricted|autoscaler)": [
```

Comment:
Not sure if this will cause an issue, but I just noticed `chibipug7501ebscsi` here

Reply:
Yep, will fix this. Should have the stack name instead.